data-protocols/ids/ids-api-transfer/src/main/java/org/eclipse/dataspaceconnector/ids/api/transfer/ArtifactRequestController.java

```diff
@@ -112,17 +113,26 @@ public Response request(ArtifactRequestMessage message) {
 
 
         // TODO this needs to be deserialized from the artifact request message
-        var destinationMap = (Map<String, Object>) message.getProperties().get(ArtifactRequestController.DESTINATION_KEY);
+        Map<String, Object> messageProperties = message.getProperties();
+        var destinationMap = (Map<String, Object>) messageProperties.get(ArtifactRequestController.DESTINATION_KEY);
```

Comment:
Can it be `DESTINATION_KEY` rather than `ArtifactRequestController.DESTINATION_KEY` here ? Same for `ArtifactRequestController.PROPERTIES_KEY` below.